TeamCode/src/main/java/org/firstinspires/ftc/teamcode/robotSubSystems/elevator/Elevator.java

```diff
@@ -0,0 +1,60 @@
+package org.firstinspires.ftc.teamcode.robotSubSystems.elevator;
+
+import static org.firstinspires.ftc.teamcode.robotSubSystems.elevator.ElevatorConstans.gearRatio;
+import static org.firstinspires.ftc.teamcode.robotSubSystems.elevator.ElevatorConstans.groundHeight;
+import static org.firstinspires.ftc.teamcode.robotSubSystems.elevator.ElevatorConstans.highHeight;
+import static org.firstinspires.ftc.teamcode.robotSubSystems.elevator.ElevatorConstans.intakeHeight;
+import static org.firstinspires.ftc.teamcode.robotSubSystems.elevator.ElevatorConstans.kP;
+import static org.firstinspires.ftc.teamcode.robotSubSystems.elevator.ElevatorConstans.lowHeight;
+import static org.firstinspires.ftc.teamcode.robotSubSystems.elevator.ElevatorConstans.midHeight;
+
+import com.qualcomm.robotcore.hardware.DcMotor;
+import com.qualcomm.robotcore.hardware.HardwareMap;
+
+import org.firstinspires.ftc.teamcode.OrbitUtils.PID;
+
+import java.util.Base64.Encoder;
+
+public class Elevator {
+    private static DcMotor firstMotor = null;
+    private static DcMotor secondMotor = null;
+    private static final DcMotor elevatorEncoder = firstMotor; //or second motor?
+    private static float height;
+    private static final PID elevatorPID = new PID(kP, 0, 0, 0, 0);
+
+    public static void init(HardwareMap hardwareMap){
+        firstMotor = hardwareMap.get(DcMotor.class, "firstMotor");
+        secondMotor = hardwareMap.get(DcMotor.class, "secondMotor");
+        //TODO reverse the motors if we need to
+    }
+
+
+    private static float getHeight(){
+        height = elevatorEncoder.getCurrentPosition() * gearRatio;
+        return height;
+    }
```

Comment:
Why is this function relevant, if it is private and only has one line?
Just update height in operate and use it instead of the function.